README.md

```diff
@@ -8,7 +8,7 @@ You can obtain BeeJU from Maven Central :
 
 # Overview
 BeeJU provides [JUnit rules](http://junit.org/junit4/javadoc/4.12/org/junit/Rule.html) that can be used to write test code that tests [Hive](https://hive.apache.org/). A JUnit rule is a means to provide resources in a test and automatically tear them down when the life cycle of a test ends.
-This project is currently built with and tested against Hive 1.2.1 but is most likely compatible with older and newer versions of Hive. The available JUnit rules are explained in more detail below.  
+This project is currently built with and tested against Hive 2.1.1 (and Hive 1.2.1) but is most likely compatible with older and newer versions of Hive. The available JUnit rules are explained in more detail below.  
```

Comment:
OK, we can remove 1.2.1 in future if we make more changes and move to newer versions of Hive.